tests/scripts/run_performance.sh

```diff
@@ -156,6 +156,7 @@ main() {
         exit 1
     fi
 
+    export TTNN_CONFIG_OVERRIDES='{"enable_fast_runtime_mode": true}';
```

Comment:
Is this going to be a permanent option?

Reply:
Yes, for a while. We do want to set it as a default option in the future

Reply:
Did you want to try putting this as a pipeline level env var in the workflow?

Reply:
@tapspatel Putting this as a workflow level env var in all perf workflows